cloudify_gcp/compute/instance.py

```diff
@@ -372,10 +371,6 @@ def create(instance_type,
     else:
         if props.get('zone', False):
```

Comment:
Can you simplify this by something like:
zone = props.get('zone') or utils.get_gcp_resource_name(gcp_config['zone']) ?